src/index.html

```diff
@@ -28,6 +28,7 @@
   <script src="https://cdn.jsdelivr.net/npm/gun/gun.js"></script>
   <script src="https://cdn.jsdelivr.net/npm/gun/nts.js"></script>
   <script src="https://cdn.jsdelivr.net/npm/gun/lib/unset.js"></script>
+  <script src="https://d3js.org/d3.v5.min.js" charset="utf-8"></script>
```

Comment:
this seems to be from another feature/pr? ;)

Reply:
I'll kick it out too, thanks for catching